src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,14 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
-  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify, autosave: true
+
+  after_initialize :init_staging_projects
+
+  def init_staging_projects
+    if new_record?
```

Comment:
Rubocop complains about not using a guard clause ...
``
return unless new_record?
...
``